docs/cli/tkgctl/management-cluster-using-clusterclass.md

```diff
@@ -6,7 +6,6 @@ Below is the sequence of management-cluster creation using clusterclass.
 1. Deploy `kapp-controller` on bootstrap cluster
 1. Deploy `providers` on bootstrap cluster
 1. Deploy `TKG meta package` to bootstrap cluster
```

Comment:
nit: name change?

Also in my earlier comment I was referring to mentioning the tkr compat config map somewhere in this sequence to call out this complication with the MC cluster bootstrap.

We can address in a followup as well.

Reply:
Updated